src/lib/SX127xDriver/SX127x.cpp

```diff
@@ -86,9 +86,9 @@ bool SX127xDriver::Begin()
   }
 
   ConfigLoraDefaults();
-  // Force the next power update
+  // Force the next power update, and use the defaults for RFO_HF or PA_BOOST
   pwrCurrent = PWRPENDING_NONE;
-  SetOutputPower(0);
+  SetOutputPower(OPT_USE_SX1276_RFO_HF ? SX127X_MAX_OUTPUT_POWER_RFO_HF : SX127X_MAX_OUTPUT_POWER);
```

Comment:
If we leave this as before, the defines can be deleted completely because they are no longer used... even more code maintenance 😄

Reply:
I did this for a reason that escapes me now 😅